packages/editor/src/components/provider/index.js

```diff
@@ -215,7 +247,7 @@ export const ExperimentalEditorProvider = withRegistryProvider(
 					}
 				);
 			}
-		}, [] );
```

Comment:
This shouldn't be removed in this PR and probably handled in a different PR to either include the missing dependencies or explore whether this is intentional for some reason..

Reply:
reverted this change